### DISCUSSION TITLE
Commandline

### DISCUSSION THREAD

**Discussion by BikerJohn66** [szpajder/dumphfdl#42] [Ideas]
The commandline for dumphfdl is getting very long. Would be possible for a new version to feed the frequencies to monitor through a textfile. When changing bands the same script can than be used with just another frequency file. Most perfect would be the samplerate to use and the frequencies.

**Comment by szpajder** [comment]
This feature doesn't need to be built into the program itself. In fact your shell already has it.

```
echo "13270 13276 13303 13312 13315 13321 13324 13342 13351" > 13MHz.txt
dumphfdl <your_common_options> $(cat 13MHz.txt)
```
